src/portal/store/types.ts

```diff
@@ -0,0 +1,17 @@
+export declare type TwinType = {
+  id: string;
+  relay: string;
+  pk: string;
```

Comment:
twin pk could be string | null.

Reply:
we should remove this, we have a store now, ill remove this